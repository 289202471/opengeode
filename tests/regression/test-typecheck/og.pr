SYSTEM NpalDsap;
    /* CIF TEXT (57, 262), (445, 263) */
    USE DataModel comment 'dataview-uniq.asn';
 
  SIGNAL MyIn(TotoT1);
    /* CIF ENDTEXT */
    CHANNEL c
        FROM ENV TO NpalDsap WITH MyIn;
    ENDCHANNEL;
    BLOCK NpalDsap;
        SIGNALROUTE r
            FROM ENV TO NpalDsap WITH MyIn;
        CONNECT c AND r;
        /* CIF PROCESS (250, 150), (150, 75) */
        PROCESS NpalDsap;
            /* CIF TEXT (0, 0), (380, 156) */
            dcl test1 TotoT1;

dcl test2 TotoT2;

dcl toto TotoT1;
            /* CIF ENDTEXT */
            /* CIF STATE (414, 17), (83, 35) */
            STATE TestState;
                /* CIF INPUT (409, 72), (92, 35) */
                INPUT MyIn(test1);
                    /* CIF TASK (391, 127), (127, 35) */
                    TASK test1 := { a toto };
                    /* CIF TASK (389, 182), (131, 35) */
                    TASK test1 := { a test2 };
                    /* CIF NEXTSTATE (420, 232), (70, 35) */
                    NEXTSTATE -;
            ENDSTATE;
        ENDPROCESS NpalDsap;
    ENDBLOCK;
ENDSYSTEM;