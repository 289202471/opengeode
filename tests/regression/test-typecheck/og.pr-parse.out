[INFO] Starting to parse model test-typecheck/og.pr
[ERROR] Failed to parse model.
    org.antlr.v4.runtime.misc.ParseCancellationException: line 3:4 no viable alternative at input '/* CIF TEXT (57, 40), (406, 38) */\n    -- the type checker must detect all type errors\n-- the code generator must complain that there is no START symbol\n    /* CIF ENDTEXT */\n    /* CIF TEXT (40, 256), (304, 53) */\n    USE DataModel comment 'dataview-uniq.asn';\n\nSIGNAL MyIn(TotoT1);\n    /* CIF ENDTEXT */\n    CHANNEL c\n        FROM ENV TO NpalDsap WITH MyIn;\n    ENDCHANNEL;\n    BLOCK NpalDsap;\n        SIGNALROUTE r\n            FROM ENV TO NpalDsap WITH MyIn;\n        CONNECT c AND r;\n        /* CIF PROCESS (202, 149), (150, 75) */\n        PROCESS NpalDsap;\n            /* CIF TEXT (0, 0), (380, 156) */\n            dcl test1 TotoT1;\n\ndcl test2 TotoT2;\n\ndcl toto TotoT1;\n            /* CIF ENDTEXT */\n            /* CIF STATE (497, 17), (83, 35) */\n            STATE TestState;\n                /* CIF PROVIDED (510, 72), (58, 23) */\n                PROVIDED TRUE;\n                    /* CIF TASK (476, 115), (127, 35) */\n                    TASK test1 := { a toto };\n                    /* CIF TASK (474, 170), (131, 35) */\n                    TASK test1 := { a test2 };\n                    /* CIF NEXTSTATE (504, 225), (70, 35) */\n                    NEXTSTATE -;\n            ENDSTATE;\n        ENDPROCESS NpalDsap;\n    ENDBLOCK;\n'
org.antlr.v4.runtime.misc.ParseCancellationException: line 3:4 no viable alternative at input '/* CIF TEXT (57, 40), (406, 38) */\n    -- the type checker must detect all type errors\n-- the code generator must complain that there is no START symbol\n    /* CIF ENDTEXT */\n    /* CIF TEXT (40, 256), (304, 53) */\n    USE DataModel comment 'dataview-uniq.asn';\n\nSIGNAL MyIn(TotoT1);\n    /* CIF ENDTEXT */\n    CHANNEL c\n        FROM ENV TO NpalDsap WITH MyIn;\n    ENDCHANNEL;\n    BLOCK NpalDsap;\n        SIGNALROUTE r\n            FROM ENV TO NpalDsap WITH MyIn;\n        CONNECT c AND r;\n        /* CIF PROCESS (202, 149), (150, 75) */\n        PROCESS NpalDsap;\n            /* CIF TEXT (0, 0), (380, 156) */\n            dcl test1 TotoT1;\n\ndcl test2 TotoT2;\n\ndcl toto TotoT1;\n            /* CIF ENDTEXT */\n            /* CIF STATE (497, 17), (83, 35) */\n            STATE TestState;\n                /* CIF PROVIDED (510, 72), (58, 23) */\n                PROVIDED TRUE;\n                    /* CIF TASK (476, 115), (127, 35) */\n                    TASK test1 := { a toto };\n                    /* CIF TASK (474, 170), (131, 35) */\n                    TASK test1 := { a test2 };\n                    /* CIF NEXTSTATE (504, 225), (70, 35) */\n                    NEXTSTATE -;\n            ENDSTATE;\n        ENDPROCESS NpalDsap;\n    ENDBLOCK;\n'
	at ee.ibk.sdl.parser.ThrowingErrorListener.syntaxError(ThrowingErrorListener.java:18)
	at org.antlr.v4.runtime.ProxyErrorListener.syntaxError(ProxyErrorListener.java:41)
	at org.antlr.v4.runtime.Parser.notifyErrorListeners(Parser.java:544)
	at org.antlr.v4.runtime.DefaultErrorStrategy.reportNoViableAlternative(DefaultErrorStrategy.java:282)
	at org.antlr.v4.runtime.DefaultErrorStrategy.reportError(DefaultErrorStrategy.java:121)
	at ee.ibk.sdl.parser.grammar.SDLParser.entity_in_system(SDLParser.java:617)
	at ee.ibk.sdl.parser.grammar.SDLParser.system_definition(SDLParser.java:383)
	at ee.ibk.sdl.parser.grammar.SDLParser.pr_file(SDLParser.java:303)
	at ee.ibk.sdl.parser.SDLParserMain.doParse(SDLParserMain.java:40)
	at ee.ibk.sdl.parser.SDLParserMain.parseFile(SDLParserMain.java:72)
	at ee.ibk.sdl.parser.SDLParserMain.main(SDLParserMain.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader.main(JarRsrcLoader.java:58)
